core/cc/posix/thread.cpp

```diff
@@ -17,6 +17,7 @@
 #include "core/cc/thread.h"
 
 #include <cstdint>
+#include <cstdlib>
```

Comment:
I cannot leave the comment on the line in question but at line 35 we should also have a free,.